src/ducks/notifications/BalanceLower/index.js

```diff
@@ -132,6 +151,10 @@ class BalanceLower extends NotificationView {
     const onlyOne = accounts.length === 1
     const firstAccount = accounts[0]
 
+    const titleKey = `Notifications.if_balance_lower.notification.${
+      onlyOne ? 'one' : 'several'
+    }.title`
```

Comment:
I see it has only been moved, but maybe we can also make it pass the grep test :D